tests/pulse/test_parametrized_evolution.py

```diff
@@ -358,3 +358,37 @@ def circuit2(params):
             jax.grad(circuit2)(params2),
             atol=5e-4,
         )
+
+    def test_mixed_device(
+        self,
+    ):
+        """Test mixed device integration matches that of default qubit"""
+        import jax
+        import jax.numpy as jnp
+
+        mixed = qml.device("default.mixed", wires=range(3))
+        default = qml.device("default.qubit", wires=range(3))
+
+        coeff = [qml.pulse.pwc(5.0), qml.pulse.pwc(5.0)]
+        ops = [qml.PauliX(0) @ qml.PauliX(1), qml.PauliY(1) @ qml.PauliY(2)]
+        H_pulse = qml.dot(coeff, ops)
+
+        @qml.qnode(default, interface="jax")
+        def qnode_def(x):
+            qml.pulse.ParametrizedEvolution(H_pulse, x, 5.0)
+            return qml.expval(qml.PauliZ(0))
+
+        @qml.qnode(mixed, interface="jax")
+        def qnode_mix(x):
+            qml.pulse.ParametrizedEvolution(H_pulse, x, 5.0)
+            return qml.expval(qml.PauliZ(0))
```

Comment:
```suggestion
        def circuit(x):
            qml.pulse.ParametrizedEvolution(H_pulse, x, 5.0)
            return qml.expval(qml.PauliZ(0))

        qnode_def = qml.QNode(circuit, default, interface="jax")
        qnode_mix = qml.QNode(circuit, mixed, interface="jax")
```
in order to facilitate reusing this and making it more stable to modifications of the circuit being used for both circuit :)

Reply:
nice thanks!